### PR TITLE
Add swift example: 'flowers' and flower images.

### PR DESCRIPTION
**Pull request by erikackermann**

### CODE REVIEW

**Comment by BrentWheeldon** [comment]
:+1:

File: Example/HTHorizontalSelectionListExample/FlowersViewController.swift [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        self.view.addConstraints(NSLayoutConstraint.constraintsWithVisualFormat("V:[selectionList]-margin-[selectedFlowerView]",
+            options: .DirectionLeadingToTrailing,
+            metrics: ["margin" : 50],
+            views: ["selectionList" : self.selectionList, "selectedFlowerView" : self.selectedFlowerView]))
+    }
+
+    // MARK: - HTHorizontalSelectionListDataSource Protocol Methods
+
+    func numberOfItemsInSelectionList(selectionList: HTHorizontalSelectionList!) -> Int {
+        return flowers.count
```
Comment (danielgrippi) [inline Example/HTHorizontalSelectionListExample/FlowersViewController.swift]: Could omit the `return` here and below since it's implicit.

**Comment by danielgrippi** [comment]
👌

File: Example/HTHorizontalSelectionListExample/FlowersViewController.swift (hunk 1 above)
Comment (erikackermann) [inline Example/HTHorizontalSelectionListExample/FlowersViewController.swift]: You can only do that with closures (and I think only with single-expression closures).

**Pull request merged**